test/e2e/start_test.go

```diff
@@ -86,6 +86,18 @@ var _ = Describe("Podman start", func() {
 		Expect(session.OutputToString()).To(Equal(name))
 	})
 
+	It("podman start single container with attach and test the signal", func() {
+		SkipIfRemote()
+		session := podmanTest.Podman([]string{"create", "--entrypoint", "sh", ALPINE, "-c", "exit 1"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		cid := session.OutputToString()
+		session = podmanTest.Podman([]string{"start", "-a", cid})
```

Comment:
nit/personal bias: I know it's not followed stringently, but I prefer that the full option to be spelled out in the test so there's no doubt what's being run.
```suggestion
		session = podmanTest.Podman([]string{"start", "--attach", cid})
```

Reply:
I will change it